pkgs/tools/networking/curlie/bump-golang-x-sys.patch

```diff
@@ -1,13 +1,15 @@
 diff --git a/go.mod b/go.mod
-index 660463a..5249458 100644
+index bebd23f..2ad575f 100644
 --- a/go.mod
 +++ b/go.mod
-@@ -2,7 +2,7 @@ module github.com/rs/curlie
- 
+@@ -1,8 +1,8 @@
+ module github.com/rs/curlie
+
  require (
- 	golang.org/x/crypto v0.0.0-20180524125353-159ae71589f3
--	golang.org/x/sys v0.0.0-20191026070338-33540a1f6037
+-	golang.org/x/crypto v0.1.0
+-	golang.org/x/sys v0.1.0
++	golang.org/x/crypto v0.0.0-20180524125353-159ae71589f3
```

Comment:
That doesn't make sense under the title `bump-golang-x-sys.patch`. Probably should drop the patch.

Reply:
it breaks without it